gitalias.txt

```diff
@@ -485,59 +513,59 @@
   add-alias = "!f() { \
       if [ $# != 3 ]; then \
         echo \"Usage: git add-alias ( --local | --global ) <alias> <command>\"; \
-        echo \"Error: this command needs 3 arguments.\"; \
```

Comment:
Please don't forget to update your base branch, so that it is at or ahead of 95d5fda20a33e64729c6fbcaae98e3f6448ab45f.

Reply:
... and as of now it should be at or ahead of 12c40fc70b37c9a26dd9f8ff72f61ca7d98e32d2.

Reply:
Unresolved this conversation, as I do still see unexpected/unrelated changes in `gitalias.txt` even after @dabrown645's merge commit.